### PR TITLE
[Bug] Staff WSOD

### PR DESCRIPTION
**Pull request by sergiogcx**
Attempts to address the WSOD in the Staff list:

### CODE REVIEW

File: moped-editor/src/components/GridTable/GridTableSearch.js [hunk 1]
```diff
@@ -324,9 +325,7 @@ const GridTableSearch = ({
               {query.config.showExport && (
                 <Button
                   disabled={
-                    projectsList &&
-                    !projectsList["project_list_view"].length &&
-                    true
```
Comment (sergiogcx) [inline moped-editor/src/components/GridTable/GridTableSearch.js:329]: It looks like we had a fixed table name under a dynamic component, it works fine under the projects list, but as this framework (GridTable and GridTableSearch) is used in other places like the Staff page, the keys would will not exist creating a WSOD.

File: moped-editor/src/components/GridTable/GridTableSearch.js [hunk 2]
```diff
@@ -324,9 +325,7 @@ const GridTableSearch = ({
...
+                    (parentData?.[query.config.table] ?? []).length === 0
```
Comment (sergiogcx) [inline moped-editor/src/components/GridTable/GridTableSearch.js:328]: So the fix is quite simple, basically pass the name of the table in question to the array, then if provided calculate its length, and if it is zero then assume it's disabled.

File: moped-editor/src/components/GridTable/GridTable.js [hunk 3]
```diff
@@ -428,7 +428,7 @@ const GridTable = ({ title, query }) => {
         {/*Toolbar Space*/}
         <GridTableToolbar>
           <GridTableSearch
-            projectsList={data}
+            parentData={data}
```
Comment (sergiogcx) [inline moped-editor/src/components/GridTable/GridTable.js:431]: Renamed to a generic type, the framework should not care what type it is.

**Review by mateoclarke — APPROVED** [review]
Good catch @sergiogcx! Instead of merging to `main`, can you point this PR agaisnt `v0.8.0-release-candidate`? 🚢

File: moped-editor/src/components/GridTable/GridTableSearch.js (hunk 1 above)
Comment (jgabitto) [reply moped-editor/src/components/GridTable/GridTableSearch.js:329]: That makes a lot of sense!

**Review by jgabitto — APPROVED** [review]
The changes you have suggested make sense as this is a reusable component!

**Pull request merged**